_source/lib/apollos.coffee

```diff
@@ -81,41 +81,75 @@ Apollos.person = (user) ->
     person = Apollos.people.findOne
       guid: new RegExp(user.personGuid, "i")
 
-
-  # we shouldnt have any code for rock in the core package
+  # TODO we shouldnt have any code for rock in the core package
   # this will probably need to adjust the rock package
 
-  # if not person and user.rock and user.rock.personId
-  #   person = Apollos.people.findOne
-  #     personId: user.rock.personId
+  # PersonGuid would fall into the same category.  The only difference is that
```

Comment:
So you want a person to have a rock guid and an apollos guid?  Why not just use the mongo id then?  This is going to complicate syncing the creation of new users who cannot have a person on rock until they add a first name and last name.  Should rock then add relationship vars to all documents that have relationships?  People -> Members -> Groups -> GroupLocations -> Locations is one large example.